sgkit/io/vcf/vcf_reader.py

```diff
@@ -429,7 +429,10 @@ def vcf_to_zarr_sequential(
     exclude_fields: Optional[Sequence[str]] = None,
     field_defs: Optional[Dict[str, Dict[str, Any]]] = None,
     read_chunk_length: Optional[int] = None,
+    check_if_done: bool = False,
 ) -> None:
+    if check_if_done and "_sgkit_parse_done" in output:
```

Comment:
This would fail if `output` was a path. Saving the flag in the mutable mapping seems a bit odd - I would expect it to be in the Zarr attributes.